draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1894,7 +1897,7 @@ cookie-string from a given cookie store.
      * The same-site-flag is "Lax" or "Default".
      * The HTTP request associated with the retrieval uses a "safe" method.
      * The target browsing context of the HTTP request associated with the
-       retrieval is a top-level browsing context.
+       retrieval is within a top-level traversable.
```

Comment:
@domenic Can I get your thoughts on this bullet point?

Reply:
I guess the most proper way would be "is the [active browsing context](https://html.spec.whatwg.org/#nav-bc) or a top-level traversable". But hopefully we will fix https://github.com/whatwg/html/issues/8463 at some point soon so you don't need these circumlocutions...

Reply:
Thanks,

Is there an ETA for that issue? I'm fine with holding off on this change until that lands

Reply:
No specific ETA, sorry.